### DISCUSSION TITLE
Invalid authentication .. Everything is right tho.

### DISCUSSION THREAD

**Discussion by Rowleys** [tuya/tuya-home-assistant#646] [General]
Hi, Really hope someone can help me 

I have install tuya v2 1.6 Im in the UK 

So when go to put my details in .. 
from Tuya IoT 

Access ID/Client ID:
Access Secret/Client Secret
Mobile app - Smartlife
Country code - +44
Them my email address & password 

then shows me "Invalid authentication"

Tuya login error response: {'code': 500, 'msg': 'system error,please contact the admin', 'success': False, 't': 1635695420728}
3:50:20 PM – (ERROR) Tuya v2 (custom integration)

**Comment by tekelectron** [comment]
Hi
Move on to Tasmota smart plugs, this integration is buggy and doesnt work propely with no support for this issue.
Thanks
